Billing.API/Models/InvoiceListItem.cs

```diff
@@ -11,12 +11,16 @@ public class InvoiceListItem
         public DateTimeOffset Date { get; }
         public string Currency { get; }
         public double Amount { get; }
+
+        [JsonIgnore]
+        public int FileId { get; }
+
         public string Filename { get; }
 
         [JsonProperty(PropertyName = "_links")]
-        public List<Link> Links { get; }
+        public List<Link> Links { get; set; } = new List<Link>();
```

Comment:
Set is not necessary.

```suggestion
        public List<Link> Links { get; } = new List<Link>();
```